onnxruntime/test/shared_lib/utils.h

```diff
@@ -52,7 +52,11 @@ void RunSession(OrtAllocator* allocator,
   auto* actual = output_tensor->GetTensorMutableData<ModelOutputT>();
   for (size_t i = 0; i != total_len; ++i) {
     if constexpr (std::is_same<ModelOutputT, float>::value || std::is_same<ModelOutputT, double>::value) {
+#ifdef USE_OPENVINO
```

Comment:
Why do we need this to migrate pipeline to github actions?

Reply:
The new machine pool has AVX512. The old one doesn't have. So the results are different.

Reply:
Do you mean that the original machine pool uses AVX2, while the newer machine uses AVX512, resulting in a larger result difference?

Reply:
Yes.

Reply:
As you can see, this PR does not change the EP's code.

Reply:
@preetha-intel Could you check if there are any accuracy issues with OpenVINO AVX512 that would require relaxation? I understand that the AVX512 saturation issue should exist for AVX2 as well. However, we can still proceed with merging it for OpenVINO EP validation for now. Just leave a comment for further review.